src/loadSchemaJSON.js

```diff
@@ -25,15 +25,19 @@ function schemaToJSON(schema, options) {
   })
 }
 
-function fetchSchemaJSON(url, options) {
-  options = options || {}
-  const graphql = options.graphql || DEFAULT_GRAPHQL
+function fetchSchemaJSON(url, extraHeaders) {
+  const headers = {
+    Accept: 'application/json',
+    'Content-Type': 'application/json'
+  }
+  for (var key in extraHeaders) {
```

Comment:
Instead of looping, it would be nicer to use either `Object.assign` or object-rest-spread, since this package specifies Node 8.10+ anyway. So I recommend:

```js
const headers = Object.assign({
  Accept: 'application/json',
  'Content-Type': 'application/json'
}, options.extraHeaders)
```

or:

```js
const headers = {
  Accept: 'application/json',
  'Content-Type': 'application/json',
  ...options.extraHeaders
}
```